include_core/omrgcconsts.h

```diff
@@ -121,6 +121,14 @@ typedef enum MM_MarkingSchemeScanReason {
 	SCAN_REASON_OVERFLOWED_OBJECT = 4, /**< Indicates the object being scanned was in an overflowed region */
 } MM_MarkingSchemeScanReason;
 
+typedef enum MM_ScavengeScanReason {
+	SCAN_REASON_SCAVENGE = 10,
+	SCAN_REASON_INCREMENTALSCAVENGE = 11,
```

Comment:
SCAN_REASON_INCREMENTALSCAVENGE is probably no different than SCAN_REASON_SCAVENGE and could be removed